cypress/support/config/settings.js

```diff
@@ -272,15 +272,15 @@ module.exports = () => ({
         environments: {
           live: {
             paths: ['/afrique'],
-            enabled: true,
+            enabled: false,
```

Comment:
@CosmicFungus  We should leave all of the live values as they were, because we still need to maintain test coverage in our live environnment until all homepages are migrated 
```suggestion
            enabled: true,
```